repositories.yaml

```diff
@@ -1309,7 +1184,7 @@ applications:
       - nimbus
     moz_pipeline_metadata_defaults:
       expiration_policy:
-        delete_after_days: 720
+        delete_after_days: 755
```

Comment:
Is this an intentional 755 instead of 775?

Increasing the retention will (should) cause the expiry alerts to start sending again for the already expiring pings so worth keeping in mind in case anyone asks